packages/block-editor/package.json

```diff
@@ -21,6 +21,7 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
+	"types": "build-types",
```

Comment:
Let's not indicate that these types should be consumed until the package's public API has a type declarations.

```suggestion
```

I'm not sure about this part, but this may also prevent other typed packages from misbehaving. I suspect if we had typed packages depending on this and provided partial definitions, we'd get a lot of errors, but I'm not sure whether that would happen or if omitting this would prevent it.

Reply:
>Let's not indicate that these types should be consumed until the package's public API has a type declarations.

Yeah, this was one of the main uncertainties I had, since it felt off to be generating / publishing incomplete types. I agree with your suggestion here 👍